source/js/ng-img-crop.js

```diff
@@ -82,7 +83,9 @@ crop.directive('imgCrop', ['$timeout', 'cropHost', 'cropPubSub', function ($time
                     }
                 }))
                 .on('area-move-end area-resize-end image-updated', fnSafeApply(function (scope) {
-                    updateResultImage(scope);
+                    if(!!!attrs.manuallyCrop) {
```

Comment:
Any particular reason for the triple negative? Or just a love of excamation marks? :)